styleguide/components/41-organisms/theme-footer/_theme-footer.scss

```diff
@@ -3,50 +3,156 @@
   @include theme('border-color', 'color-primary--lighten-4', 'theme-footer-border-bottom-color');
 
   width: calc(100% + 2 * #{$gutter-width});
-  margin-top: 4rem;
-  margin-right: -$gutter-width;
-  margin-left: -$gutter-width;
-  padding: 2.7rem 0 2.1rem;
+  margin-right: -1rem;
+  margin-left: -1rem;
+  padding: 0;
   border-bottom: 2px solid;
 
   + .footer {
     margin-top: 0;
   }
 
-  .content-container {
-    display: flex;
-    flex-direction: column;
-    padding: 0 $gutter-width;
+  .contact {
+    .icon-list {
+      margin: 0;
+
+      li:last-of-type {
+        margin-bottom: 0;
+      }
+
+      @include tablet {
+        column-gap: gutter();
+        column-count: 2;
+      }
+    }
+  }
+
+  .social {
+    text-align: center;
 
     @include desktop {
-      padding: 0;
+      text-align: left;
+
+      &:only-child {
+        text-align: center;
+      }
     }
 
     h3 {
-      align-self: center;
+      max-width: 100%;
     }
 
     .social-list {
-      align-self: center;
+      display: inline-block;
+      width: auto;
+      margin: auto;
 
-      @include tablet {
-        max-width: 50%;
+      ul {
+        justify-content: flex-start;
+        width: calc(100% + 1rem);
+        margin: 0 -.5rem;
+
+        li {
+          padding: .5rem;
+        }
+
+        a {
+          &,
+          &[download]:not(.button),
+          &[href^="http://"]:not(.button),
+          &[href^="https://"]:not(.button) {
+            @include theme('background-color', 'color-secondary', 'theme-footer-social-media-icon-background');
+          }
+        }
       }
+    }
+  }
 
-      @include desktop {
-        max-width: 30%;
+  .newsletter {
+    border-radius: 0;
+    font-size: .8rem;
+
+    .inner-box {
+      padding: 0;
+      background-color: transparent;
+
+      &::before {
+        content: none;
       }
+
+
+
+      @include tablet {
+        min-height: 7rem;
+        padding-left: 9rem;
+
+        &::before {
+          @include spot-image('letter-light', 7rem);
+        }
+
+        &::before {
+          top: 0;
+          right: auto;
+          bottom: auto;
+          left: 0;
+        }
+      }
+    }
+
+    .subscribe {
+      max-width: calc(#{$bp-max-content} / 12 * 5);
```

Comment:
Maybe add some more comments to pieces of code like this to document the reasoning behind these calc() functions?

Reply:
It's because susy is relative to it's direct container, not to the content-container width.
Comment added.